### DISCUSSION TITLE
Last 3.1 Version

### DISCUSSION THREAD

**Discussion by wardsimon** [SpinW/spinw#79] [General]
SpinW will be moving to SpinW4 in the future. This is the last stable build before the new version is released (Though there will be hot fixes if required.)

<hr /><em>This discussion was created from the release <a href='https://github.com/SpinW/spinw/releases/tag/v3.1.2'>Last 3.1 Version</a>.</em>

**Comment by pastle** [comment]
Will SpinW4 support the python version?
